Sentinel/Classes/Core/ToolTableSection.swift

```diff
@@ -20,11 +19,18 @@ public class ToolTableSection: NSObject {
     let items: [ToolTableItem]
 
     // MARK: - Lifecycle
-    
-    /// Creates an instance of the section.
+
     public init(title: String? = nil, items: [ToolTableItem]) {
         self.title = title
         self.items = items
-        super.init()
+    }
+}
+
+// MARK: - Identifiable conformance
+
+extension ToolTableSection: Identifiable {
+
+    public var id: String {
+        title ?? UUID().uuidString
```

Comment:
Due to the nature of computed property and if the `title` is `nil`, the returned string will be different every time the `id` is called/used.

Reply:
Nice catch 👌 
I added it as a property which will be instantiated through the init